edu/wisc/ssec/mcidasv/resources/python/background.py

```diff
@@ -496,6 +518,33 @@ def getDefaultLayerLabel(self):
         return defaultLabel
 
 
+def _dictFromAttributeList(attrList):
+    """Build a attrName->attrValue dictionary from a list of NetCDF Attributes."""
+    result = dict()
+    for attr in attrList:
+        result[attr.getFullName()] = _getNetcdfAttributeValue(attr)
+    return result
+
+
+def _getNetcdfAttributeValue(attr):
+    """Helper to return netcdf Attribute value as either numeric or string value."""
+    if attr.isArray():
+        # umm. convert to python list i guess?
```

Comment:
I'd say that the list conversion is probably the right thing to do, especially given that Jython allows users to treat a given Java collection like its Python equivalent.